definitions/tools/vep.cwl

```diff
@@ -90,6 +88,10 @@ inputs:
         inputBinding:
             prefix: "--fasta" 
             position: 8
+    plugins:
+        type: string[]?
+        inputBinding:
+            prefix: "--plugin"
```

Comment:
To get the `--plugin` prefix before each plugin, this will need to be something like:
```cwl
    type:
      type: array
      items: string
      inputBinding:
        prefix: "--plugin"
```

See also [this page](https://www.commonwl.org/user_guide/09-array-inputs/) for examples of three different ways to handle array inputs.

Reply:
Good catch!